main.nf

```diff
@@ -124,7 +125,7 @@ Channel
     .groupTuple(sort: true)
     .set { read_files }
```

Comment:
Can replace this `Channel` with a simpler one _(see previous comment, point 1.)_